server/pbenchinacan/load_keycloak.sh

```diff
@@ -14,25 +14,34 @@
 # The script defaults to using master realm username/password as admin/admin
 # unless specified otherwise by 'ADMIN_USERNAME' and 'ADMIN_PASSWORD' env
 # variables. The script also defaults the keycloak redirect URI as
-# "https://localhost:8443/*" unless specified otherwise by 'KEYCLOAK_REDIRECT_URI'
+# "https://<host_ip>:8443/*" unless specified otherwise by 'KEYCLOAK_REDIRECT_URI'
 # env variable.
 
-KEYCLOAK_HOST_PORT=${KEYCLOAK_HOST_PORT:-"http://localhost:8090"}
-KEYCLOAK_REDIRECT_URI=${KEYCLOAK_REDIRECT_URI:-"https://localhost:8443/*"}
+if [[ -z "${PB_HOST_IP}" ]]; then
+    host_ip_list=$(hostname -I)
+    PB_HOST_IP=${host_ip_list%% *}
+    export PB_HOST_IP
+fi
+
+KEYCLOAK_HOST_PORT=${KEYCLOAK_HOST_PORT:-"https://localhost:8090"}
+KEYCLOAK_REDIRECT_URI=${KEYCLOAK_REDIRECT_URI:-"https://${PB_HOST_IP}:8443/*"}
 ADMIN_USERNAME=${ADMIN_USERNAME:-"admin"}
 ADMIN_PASSWORD=${ADMIN_PASSWORD:-"admin"}
 # These values must match the options "realm" and "client in the
 # "openid" section of the pbench server configuration file.
 REALM=${KEYCLOAK_REALM:-"pbench-server"}
 CLIENT=${KEYCLOAK_CLIENT:-"pbench-client"}
 
+PB_DEPLOY_FILES=${PB_DEPLOY_FILES:-"/var/tmp/pbench/pbench_server_deployment"}
+
 end_in_epoch_secs=$(date --date "2 minutes" +%s)
 
 # Run the custom configuration
 
 ADMIN_TOKEN=""
 while true; do
-  ADMIN_TOKEN=$(curl -s -f -X POST "${KEYCLOAK_HOST_PORT}/realms/master/protocol/openid-connect/token" \
+  ADMIN_TOKEN=$(curl -s -f -X POST  --cacert "${PB_DEPLOY_FILES}/pbench-server.crt" \
```

Comment:
I don't think we need this change in order to run functional tests, because `run-server-func-tests` already [defines `CURL_CA_BUNDLE`](https://github.com/distributed-system-analysis/pbench/blob/ef8a33da0670b4ba8b21faccd0b1ee3c1f6c7664/jenkins/run-server-func-tests#L23) which should take care of this (assuming that this `pbench-server.crt` was signed by the Pbench CA...).

So, when do we run this script other than under `run-server-func-tests`?  Perhaps what we should be doing is setting up `CURL_CA_BUNDLE` for those cases, rather than adding all these `--cacert`'s here.